server/server.go

```diff
@@ -399,6 +399,14 @@ func New(
 
 	// Prepare streaming server
 	streamServerConfig := streaming.DefaultConfig
+	if config.StreamIdleTimeout != "" {
+		idleTimeout, err := time.ParseDuration(config.StreamIdleTimeout)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse timeout as duration")
```

Comment:
There is no format included and we'd like to stick to the `github.com/pkg/errors` package rather than `fmt`:
```suggestion
			return nil, errors.New("unable to parse timeout as duration")
```